### ISSUE TITLE
Mouse Offset Incorrectly When Borderless Gaming 8.4 used

### ISSUE THREAD

**Issue by Firedingo** [Codeusa/Borderless-Gaming#242]
So I saw someone else a few years ago had an issue with Civ 5 and this issue under borderless gaming v 7.5 I think. Specifically Civ 5, Banished and possibly a few other games when forced into a borderless fullscreen mode have the mouse visually offset to where it actually is. I noticed depending on what resolution you selected in the game it could make this more or less of an issue. For example when I choose 1920x1080 it's barely off but if I choose 1600x900 it visually looks the same because the aspect ratio matches but the cursor is like half a screen size offset.
I'm not sure if this is something you can fix or if it's an issue with the way the game has been coded. I'm running Borderless Gaming v 8.4 with updated drivers and game versions.

**Comment by MuffinCupcake** [comment]
Got this problem as well.

**Comment by Acecool** [comment]
I have this issue in some games that do not support dynamic resizing of the window... This happens because the game sets the size when it starts and when the game is resized ( when you remove the title-bar and borders, that adds more game-panel viewing area pixels / size but the game doesn't update the game-panel size when it changes meaning the offset is incorrect ) it shifts the clicks...

This has to do with each individual game; but there should be an option added so if the offset issue happens, BorderlessGaming could add a buffer between the mouse-clicks, ie it can capture the mouse-clicks and then translate the x-y position to a new position and then apply the click...

**Comment by Firedingo** [comment]
Agreed that the game is responsible. Unlikely old games will get the update they need so I also support BorderlessGaming having an option to compensate for the game.

**Comment by AGRossau** [comment]
I know this is an old issue, but as it is still open, I thought I would chime in. Also for anyone else who - like me - finds this due to having similar issues.
I don't know if it will help in your case, but my mouse was offset in The Witcher Enhanced Edition, and when I disabled Auto-Maximize that fixed it.

**Comment by Adriaseth** [comment]
For people stumbling upon this. What fixed it for me was to have the game and the program open already. then right click the Windowed Borderless Gaming icon in taskbar and go to "Settings>Game Settings". In the screen that opens under "Size & Offset" input the screen resolution you want to use in the "Resolution" which was 1920 x 1080 for me. Apply it and it should solve your issue!

**Comment by rbpaiett** [comment]
I just go to the game settings and click 'Apply'.  Don't have to change any settings if you already have done that.  A minor pain but until there's a fix.  Sometimes other processes kind of distract WBG from doing it's thing.

**Comment by CoolBeanie** [comment]
I think I figured out how to fix this. Windows was scalling applications to 125%. So I right clicked, went into properties, Compatibility, Change high DPI settings, Override high DPI scaling behavior Scaling performed by: Application. I believe with this it scales the application to 100% instead of 125% so the cursor isn't offset.

**Comment by rbpaiett** [comment]
Did you do this for Windowed Borderless or for the game WBG is modifying?

**Comment by Louna2138** [comment]
@rbpaiett I changed the global Scale to 100% in Windows 10 Settings > System > Display > Scale and layout 
![2021-12-01_00h10_26](https://user-images.githubusercontent.com/13695107/144136246-38a29ec2-7947-4184-bfda-907ab575c8a3.png)

**Comment by nonplayer** [comment]
Having this issue as well. Sadly my monitor is already at 100%, so the scaling fix didn't apply.

**Comment by CoolBeanie** [comment]
Yeah you could scale for 100% for all of windows but I only did it for the particular application. I basically right clicked on the application's exe to get it to work in compatibly mode. As windows in 125% tries to scale up the application which is what causes the mouse offset.
![Screenshot](https://user-images.githubusercontent.com/94817412/156850820-18325dee-ce63-494e-a033-983f4d41cb6f.png)

**Comment by dcraig327** [comment]
> I know this is an old issue, but as it is still open, I thought I would chime in. Also for anyone else who - like me - finds this due to having similar issues.
> 
> I don't know if it will help in your case, but my mouse was offset in The Witcher Enhanced Edition, and when I disabled Auto-Maximize that fixed it.

Thank you! I forgot there were per-game options and disabling Auto-Maximize also fixed the offset for Witcher Enhanced Edition for me as well.

[Love & Peace](https://www.youtube.com/watch?v=e5WUVwR4__o)

**Comment by Noviath** [comment]
I'm with @nonplayer, right now using BG 9.5.6 and trying to run star citizen and BG does a good job filling the screen but mouse offset still there and non of the solutions here worked for me aswell :(

**Comment by Aezriel82** [comment]
can confirm this issue still in version 10. 

im playing diablo4 on a 16:9 48inch oled. i am running in a window to create a more widescreen experience 3840x1800, as it in game gives u substantially a larger viewing POV, the mouse offset seems to be exactly the size of the titlebar and right border. as the title goes away, ingame screen SNAPS up to the top right position the title used to be..but the game mouse postion stays as it should. surely there is a way to prevent this snap from occuring. 

it does not seem to happen when the window is in fullscreen ?? 

i have seen another border removing tool mention the following, that may help? probs dead end thou:

( using 0xcf0000 and 0x20301 work best for hiding borders in fullscreen multimedia windows, but might cause graphical UI weirdness when applied to regular windows. For regular windows the values 0xcb0000 and 0x20300 are recommended instead. ) - - i have no clue if this is at all a solution and have no clue how to implement it even if it might be related... hoping someone smarter might be able to confirm?

**Comment by filevans** [comment]
hello, could a mouse offset be added? as i get this problem on some games too, in the menu, and you can't press the menu item
